data/News/2021-05-19-volunteer-roles.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Seeking event volunteers
+published: 2021-05-18
+---
+
+As we start thinking ahead to our on the day event logistics, PyConline is actively seeking volunteers for the following roles: Green Room Tech Checker, and Show Producer. 
+
+If you are interested in either of these roles, please get in touch at [contact@pycon.org.au](mailto:contact@pycon.org.au)
+
+Please note that these roles are timezone dependent, and require someone who can work within the UTC+10 (Australia/Melbourne) timezone. 
+
+
+### Green Room Tech Checker
+
+Availability: from mid-July 2021, through event date.
+
+Making sure our speakers look and sound good.
+
+ * Ideally AV savvy, but some training can be provided.
+ * Work presenters through a supplied AV checklist.
+ * Answer any non-AV questions about the event.
+
+
+### Show Producer
+
+Availability: immediate, through event date.
+
+The Producer ultimately makes sure the show goes on. The Producer ensures that the program selected and scheduled by the Program Chair actually happens!
+
+This role is part of the core organiser team and should be availabile for weekly co-ordination syncs with the core team, and asynchronously by the organiser discord. 
+
+ * Responsible for run-lists, user moderation of backstage.
+ * Point of contact and volunteer wrangler for session hosts and speaker wranglers.
+ * Keep session hosts and wranglers organised, make sure session hosts have introductions for each presenter, etc
+ * Make ultimate decisions about show content on the day of the event. 
```

Comment:
```suggestion
 * Make ultimate decisions about show content on the day of the event. 
 
Extensive support and mentorship will be provided to our show producer. We're asking the volunteer to join our core team primarily so that they experience of the inner workings of the conference team prior to the event/stream.
```

Reply:
already added above the list